dataset/include/scipp/dataset/map_view.h

```diff
@@ -261,8 +261,8 @@ template <class Base, class Access> class MutableView : public Base {
   }
 };
 
-SCIPP_DATASET_EXPORT Variable
-masks_merge_if_contains(const MasksConstView &masks, const Dim dim);
+SCIPP_DATASET_EXPORT Variable irreducible_mask(const MasksConstView &masks,
```

Comment:
As far as I can see we don't test this

Reply:
Indeed, there appear to be only indirect tests. I will do this as part of the follow up I am working on (see #1119).